go-controller/pkg/ovn/egressqos_test.go

```diff
@@ -900,6 +949,8 @@ var _ = ginkgo.Describe("OVN EgressQoS Operations", func() {
 				nodeSwitch,
 			}
 			gomega.Eventually(fakeOVN.nbClient).Should(libovsdbtest.HaveDataIgnoringUUIDs(expectedDatabaseState))
+			// Ensure default EgressQoS object is updated with zone success status.
```

Comment:
all your tests are zone successes, please add a test for failure.
also it seems like we are testing only 1 zone? what if I have two?

Reply:
good catch ! added test for failure case.
we have an unit test [here](https://github.com/ovn-org/ovn-kubernetes/pull/4158/files#diff-b6ec0f15cf426791db8d26133bef7c248916d4873573c69b07d1409111ebcccbR345-R363) in cluster manager covering two zones, is that ok ?